### PR TITLE
[zh] Update configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update configure-upgrade-etcd.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -672,3 +672,20 @@ ETCDCTL_API=3 etcdctl --data-dir <data-dir-location> snapshot restore snapshotdb
...
+## Upgrading etcd clusters
+
+For more details on etcd upgrade, please refer to the [etcd upgrades](https://etcd.io/docs/latest/upgrades/) documentation.
+
+{{< note >}}
+Before you start an upgrade, please back up your etcd cluster first.
+{{< /note >}}
+-->
+## 升级 etcd 集群
+有关 etcd 升级的更多详细信息，请参阅 [etcd 升级](https://etcd.io/docs/latest/upgrades/) 文档。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md:687]:
```suggestion
有关 etcd 升级的更多详细信息，请参阅 [etcd 升级](https://etcd.io/docs/latest/upgrades/)文档。
```

File: content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -672,3 +672,20 @@ ETCDCTL_API=3 etcdctl --data-dir <data-dir-location> snapshot restore snapshotdb
...
+{{< note >}}
+在开始升级之前，请先备份您的 etcd 集群。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md:689]:
```suggestion
在开始升级之前，请先备份你的 etcd 集群。
```

Comment (my-git9) [reply content/zh/docs/tasks/administer-cluster/configure-upgrade-etcd.md]: Adjusted,thanks for your review

**Pull request merged**